releases/net472/dotnet472-changes.md

```diff
@@ -125,6 +125,8 @@
 * Fixed error connecting to read only secondary via routing table when connecting using the AOAG listener and specifying Column Encryption Setting=Enabled and ApplicationIntent=Readonly. [510851, System.Data.dll, Bug]
 * Integrating Azure AD Universal and Multi-factor authentication (MFA) support with SQL .NET driver. [521874, System.Data.dll, Feature]
 * Signature verification results are now cached improving query execution times for enclave-based queries. [552762, System.Data.dll, Feature]
+* Fixes an issue where while using SqlConnection.ConnectionString to set a null or empty connection string a NRE exception would be thrown by the usage of the API while using .Net Framework 4.7.2. [611802, System.Data.dll, Bug]
+* While connecting to Azure SQL DB and using MultipleActiveResultSets=true in the connection string with System.Data.SqlClient.SqlConnection, the async query operations would lead to bad TDS protocol request stream to be sent from the client, causing the Async Query APIs to fail. This is now fixed. [620109, System.Data.dll , Bug]
```

Comment:
A somewhat clearer first sentence: "When using System.Data.SqlClient.SqlConnection to connect to Azure SQL DB with MultipleActiveResultSets=true in the connection string,"
the async query operations --> async query operations
would lead to bad --> result in a bad 
to be sent --> being sent
Async Query APIs --> lowercase (async query APIs)

Reply:
Will fix last 3